### PR TITLE
Styling pages: signup, login, home, posts

### PR DESCRIPTION
**Pull request by heykathl**
## Before you start

1. Have you run `git pull origin main` on your branch to bring in the latest changes?
2. Is there an issue on the repository for this request? If not, please create one.
3. Have you followed the correct workflow in the [team documentation](https://docs.google.com/document/d/13yi709ax4nmfTG9J0F57yEFpYRH7fdtczwvVJCrNIsM/edit#)?
4. Conflict resolution to be dealt with **on GitHub**.

## What do these code changes do?
Styling signup, login, home, posts pages
## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [ ] I have added thorough tests.
- [ ] I have created a PR ticket on the [Trello board](https://trello.com/b/P8lEsvZe/acebook).
- [ ] I will link the PR to the Trello card once created.
- [x] I will send the PR in the Slack channel and ping @here.

### CODE REVIEW

**Review by PKilgarriff — APPROVED** [review]
I'm onboard - especially with the slightly jumping buttons. If we get a chance to do some more Koala imagery there are also good places for them to go

File: views/posts/index.hbs [hunk 1]
```diff
@@ -25,28 +25,24 @@
     <div class="post-container card" id="{{this._id}}">
       <div class="actual-post">
         <div class="profile-picture">
-          <a href="/profile/{{this.user._id}}">
```
Comment (clovellbsc) [inline views/posts/index.hbs:28]: is this not taking away the link to the person's profile?

File: views/posts/index.hbs [hunk 2]
```diff
@@ -25,28 +25,24 @@
...
+          <img
+            src={{this.user.profilePicture}}
+            width="100"
+            height="100"
+            class="rounded-circle"
+          />
         </div>
         <div class="post-body">
           <div class="post-header">
-            <a href="/profile/{{this.user._id}}">
```
Comment (clovellbsc) [inline views/posts/index.hbs:39]: same here?

**Review by clovellbsc — CHANGES_REQUESTED** [review]
just a couple of things with the anchor tags removing links to the profile but otherwise, nice work! 😊

File: views/posts/index.hbs (hunk 1 above)
Comment (PKilgarriff) [reply views/posts/index.hbs:28]: Good spot

File: views/posts/index.hbs (hunk 2 above)
Comment (heykathl) [reply views/posts/index.hbs:39]: Yes I've re-added them in now, thanks for spotting it

**Review by clovellbsc — APPROVED** [review]

**Review by adamwoodcock98 — APPROVED** [review]
Amazing work, looks so much better!

**Pull request merged**